lms/static/sass/course/_info.scss

```diff
@@ -56,6 +56,95 @@ div.info-wrapper {
       font-style: normal;
     }
 
+    div.upgrade-banner {
+      // This banner uses the Pattern Library's defined variables
+      background-color: $white;
+      border: 1px solid $uxpl-gray-background;
+      @include border-left(0px);
+      width: 100%;
+      display: table;
+
+      .green-border {
+        width: 10px;
+        height: 100%;
+        margin: 0;
+        vertical-align: top;
+        background: $uxpl-green-base;
```

Comment:
Use semantic variables and not the raw colors, because again this will be very hard to theme.